aws_lambda_builders/actions.py

```diff
@@ -138,24 +144,48 @@ def execute(self):
             utils.create_symlink_or_copy(str(source_path), str(destination_path))
 
 
+class LinkSinglePathAction(BaseAction):
+    NAME = "LinkSource"
+
+    DESCRIPTION = "Creates symbolic link at dest, pointing to source"
```

Comment:
Complete nit: let's use full forms for description.

Reply:
Do we show this description for the customers any where ?

Reply:
> Complete nit: let's use full forms for description.

What do you mean by this @sriram-mv ?

> Do we show this description for the customers any where ?

I looked into this and actually I don't think so.

Reply:
dest -> destination.

Reply:
ohhh I thought you were referring to a form, as in filling out a form, or something. Yeah I can change it (although again I'm not sure what this description is used for as I don't think it's displayed anywhere AFAIK)